packages/@aws-cdk/aws-kinesisfirehose-alpha/test/integ.delivery-stream.js.snapshot/aws-cdk-firehose-delivery-stream.template.json

```diff
@@ -252,6 +252,9 @@
    "ca-central-1": {
     "FirehoseCidrBlock": "35.183.92.128/27"
    },
+   "ca-west-1": {
+    "FirehoseCidrBlock": "40.176.98.192/27/27"
```

Comment:
Should be `40.176.98.192/27`. There is an extra `/27` at the end.

Reply:
Nice catch

Reply:
Not sure why but this keeps getting automatically updated with the extra `/27` and overriding my changes.

Reply:
Realized I was not updating the value in the correct place. It should be fixed now.